Tests/StreamChatUITests/SnapshotTests/CommonViews/ChatChannelHeaderView/ChatChannelHeaderView_Tests.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+final class ChatChannelHeaderView_Tests: XCTestCase {
```

Comment:
Would be nice to add unit tests to the missing subtitleText coverage: 
- WhenIsDirectMessageChannel_AndIsMemberOnline
- WhenIsDirectMessageChannel_AndIsLastActive
- WhenIsDirectMessageChannel_AndIsOffline
<img width="1072" alt="Screenshot 2022-04-12 at 00 43 27" src="https://user-images.githubusercontent.com/12814114/162851551-8daf7627-e406-43d7-95b5-7508330bd444.png">

Reply:
Tests added